schema/layout.mod

```diff
@@ -30,6 +30,16 @@
 	not the local tenths as adjusted by the staff-size element.
 -->
 
+<!-- Entities -->
+
+<!--
+	The common-layout entity specifies the sequence of page,

```

Comment:
The layout entity from the first version of this pull request duplicated the layout entity defined in partwise.dtd and timewise.dtd. The renaming to common-layout avoids this duplication, which could cause XML parser fatal errors.